tests/attr/test_saliency.py

```diff
@@ -36,6 +40,25 @@ def _get_multiargs_basic_config() -> Tuple[
     return model, inputs, grads, additional_forward_args
 
 
+def _get_multiargs_basic_config_large() -> Tuple[
```

Comment:
nit: It might be good to add a GPU / DataParallel test with this functionality if possible, it should only require adding a new config similar to these existing NoiseTunnel configs with this new parameter. https://github.com/pytorch/captum/blob/master/tests/attr/helpers/test_config.py#L282

Reply:
Good point! I'll add DP tests.